### ISSUE TITLE
Calculators FFY and FFX

### ISSUE THREAD

**Issue by Sondr11** [J35P312/AMYCNE#5]
Hi
I have looked at the code to calculate FFY and FFX, but I don't quite understand how to calculate it.  Can you describe in detail how it is calculated?

**Comment by J35P312** [comment]
Hello!
First the coverage of each bin is normalised as written here (PMID: 29579039). Next, the bins are filtered further: removing bins of low mapping quality, or bins of high coverage on Y (> 50% compared to autosomal coverage).

Lastly, the median of the normalised coverage on Y in the remaining bins are computed, and scaled according to the user defined scaling factor.

**Comment by Sondr11** [comment]
Thank you for answer.
I used amycne for FFY and FFX calculations with NIPT.  I have built a coefficient suitable for me.  correlation coefficient with 1 method (I consider it standard) r=0.86.  I am quite satisfied using AMYCNE.  I am currently looking for a way to use FFY and FFX for the prognosis of Syndrome XXY and XYY.  Basically XXY, I understand that FFX is close to 0, XYY, the ratio FFY/FFX = 2.  However, I'm having trouble determining XYY.  Is there any other formula to determine XYY?
